tethys_gizmos/gizmo_options/map_view.py

```diff
@@ -263,12 +266,15 @@ class MVDraw(SecondaryGizmoOptions):
         drawing_options = MVDraw(
             controls=['Modify', 'Delete', 'Move', 'Point', 'LineString', 'Polygon', 'Box'],
             initial='Point',
-            output_format='WKT'
+            output_format='WKT',
```

Comment:
Please use snake case for arguments and variables (underscores) rather than camel case.